plugins/available/cdup.plugin.bash

```diff
@@ -0,0 +1,17 @@
+cite about-plugin
+about-plugin 'cdup - traverse parent directories with ease. inspired by a script from tyskby'
+
+cdup() {
+  if [[ $1 =~ .{2,} ]]; then
```

Comment:
Could you please add the additional descriptive tags in here to describe how the function works and what the parameters are? See here for an example: https://github.com/Bash-it/bash-it/blob/master/plugins/available/dirs.plugin.bash#L71

    about 'save a bookmark'
    param '1: bookmark name'
    example '$ S mybkmrk'
    group 'dirs'